### DISCUSSION TITLE
Εγκατάσταση Arch Linux σε εξωτερικό μέσο ή εικονική μηχανή

### DISCUSSION THREAD

**Discussion by p14kairas** [courses-ionio/hci#1592] [Announcements]
Θα ήθελα να ρωτήσω εάν για τις ασκήσεις εντολών θα μπορούσαμε να τρέξουμε το Arch Linux live από στικάκι κατευθείαν αντί να το κάνουμε ολόκληρο install ή έστω μέσω virtual machine γιατί μπορεί να προκληθούν πολλά προβλήματα στο κύριο σύστημα.?

**Comment by epidrome** [comment]
οι οδηγίες έχουν την απάντηση:
https://github.com/courses-ionio/projects/blob/master/dokey/index.md

η απόκληση από τις οδηγίες βαθμολογείται θετικά ή αρνητικά ανάλογα με την τεκμηρίωση και το πόσο προσθέτει-αφαιρεί γνώσεις-δεξιότητες
